src/components/Todo/Todo.js

```diff
@@ -4,25 +4,23 @@ import Checkbox from 'components/Checkbox/Checkbox'
 import { FaTrashAlt } from 'react-icons/fa'
 import style from './todo.module.scss'
 
-const Todo = ({ todo, index, handleDeleteIconClick }) => {
+const Todo = ({ todo, index, handleDeleteIconClick, onStatusChange, status }) => {
```

Comment:
Para que sea menos confuso, cambia la prop `todo` por `title`. 

Por otro lado, las props nunca son `handle` (handler es lo que le asignamos), y en este caso lo que importa es la acción del todo, no donde se hace, por lo que podés cambiarlo para que sea más semántico, como `onDelete` (lo que asignás a esa prop es la función que se ejecuta cuando un todo se borra, cómo borrás ese todo o mediante qué acción no le importa al componente).


```suggestion
const Todo = ({ index, title, status, onDelete, onStatusChange }) => {
```